plugin/rewrite/class.go

```diff
@@ -34,6 +35,7 @@ func (rule *classRule) Rewrite(ctx context.Context, state request.Request) Resul
 	if rule.fromClass > 0 && rule.toClass > 0 {
 		if state.Req.Question[0].Qclass == rule.fromClass {
 			state.Req.Question[0].Qclass = rule.toClass
+			RequestRewriteClassCount.WithLabelValues(metrics.WithServer(context.Background()), string(rule.fromClass), string(rule.toClass))
```

Comment:
Please Use the context which is passed to `Rewrite` method instead of creating new ones. Also the same for the following cases.

Reply:
good catch. this was a copy paste error from another plugin.